README.md

```diff
@@ -59,16 +59,18 @@ Getting started
 For iCE40 support, install [Project IceStorm](https://github.com/YosysHQ/icestorm) to `/usr/local` or another location, which should be passed as `-DICESTORM_INSTALL_PREFIX=/usr` to CMake. Then build and install `nextpnr-ice40` using the following commands:
 
 ```
-cmake . -DARCH=ice40
+mkdir -p build && cd build
+cmake .. -DARCH=ice40
+make nextpnr-all-bba
```

Comment:
I don't think this is needed?

Reply:
It was not needed before, but now it fails if you run it with -j parameter even for ice40

Reply:
@gatecat found solution for others, but himbaechel is already having issue that it starts bba creation twice if we run with -j
Next change will work for rest of archs.

```
diff --git a/CMakeLists.txt b/CMakeLists.txt
index c3561e3c..1f68a62f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -337,6 +337,8 @@ function(add_nextpnr_architecture target)
 
     add_library(nextpnr-${target}-chipdb INTERFACE)
 
+    add_dependencies(nextpnr-${target}-chipdb nextpnr-${target}-bba)
+
     target_link_libraries(nextpnr-${target}-core INTERFACE nextpnr-${target}-chipdb)
 
     # CLI/GUI runner
```

Reply:
anyway I have removed this line to prepare README with usage as it should be

Reply:
I thought I fixed this by removing SERIALIZE_CHIPDBS but evidently not... I can't spend time on this for a bit, but once I can I'd really like to review it in depth so it doesn't reoccur later somehow.

Reply:
Should be fixed by https://github.com/YosysHQ/nextpnr/pull/1444